src/python/pants/backend/helm/subsystems/BUILD

```diff
@@ -1,6 +1,27 @@
 # Copyright 2022 Pants project contributors (see CONTRIBUTORS.md).
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
 
-python_sources()
+python_requirement(
+    name="yamlpath",
+    requirements=[
+        "yamlpath>=3.6,<3.7",
+        "ruamel.yaml>=0.15.96,!=0.17.0,!=0.17.1,!=0.17.2,!=0.17.5,<=0.17.21",
+    ],
+    resolve="helm-post-renderer",
+)
 
-python_tests(name="tests")
\ No newline at end of file
+python_sources(
+    sources=["*.py", "!*_test.py", "!post_renderer_main.py"],
+    dependencies=[":post_renderer"],
+)
+
+python_sources(
+    name="__post_renderer_mainr",
+    sources=["post_renderer_main.py"],
+    resolve="helm-post-renderer",
+    skip_mypy=True,
```

Comment:
Disabled beacuse I could not find any type-stubs for the `yamlpath` package. However if there is any other way of making mypy ignore the `yamlpath` imports but still do type-checking in the rest it would be great.